infrastructure/microvm/cloudhypervisor/create.go

```diff
@@ -119,6 +119,8 @@ func (p *provider) buildArgs(vm *models.MicroVM, state State, _ *logrus.Entry) (
 	}
 	args = append(args, "--disk", "path="+rootVolumeStatus.Mount.Source)
 	args = append(args, fmt.Sprintf("path=%s,readonly=on", state.CloudInitImage()))
+	args = append(args, "--fs", fmt.Sprintf("tag=user,socket=%s,num_queues=1,queue_size=1024", state.VirtioFSPath()))
```

Comment:
Guessing we only need this if we are using virtiosfs. We should probably have a check and only add this arg if required.